unit_tests/src/test_process_stash.cpp

```diff
@@ -0,0 +1,68 @@
+#include "catch.hpp"
+
+#include "process_stash.hpp"
+
+#include <boost/any.hpp>
+
+#include <algorithm>
+#include <iterator>
+#include <string>
+
+TEST_CASE("process_stash basic operations")
+{
+    // A POD type used to demonstrate the stash's ability to store heterogeneous data.
+    struct pod
+    {
+        int x = 0;
+        int y = 0;
```

Comment:
by heterogeneous... you mean... what?

there are both ints?    can you put different types here at the same time?

Reply:
I think it's referring to the stash, not the struct.

Reply:
ohhh, found a parser :)

okay, so maybe a slight rephrasing of line 13 is in order...

Reply:
Will do. I felt that line would need a tweak.

Reply:
Updated.